pkg/kritis/review/review.go

```diff
@@ -52,12 +52,26 @@ func New(client metadata.Fetcher, c *Config) Reviewer {
 	}
 }
 
+// Flatten allowlist from a list of generic attestation policies
+func generateGapAllowlist(gaps []v1beta1.GenericAttestationPolicy) []string {
```

Comment:
This should really take a single GAP, as it doesn't make much sense to have a single flattened allowlist from multiple policies.  On the other hand, I understand that there are some problems with this, please see my other comment below.